scripts/test_motor_board.py

```diff
@@ -38,7 +38,7 @@
 import time
 import serial
 import string
-import thread
+import _thread
```

Comment:
We should migrate to the new threading module instead of using the Python2 compat mode.